classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    student.classroom = self
+    @students << student
+  end
```

Comment:
- [ ] I think it would be great if you add the classroom as well in the student class since as a part in the requirements you should add his association so kindly add it since this method serves as the 'belongs-to' side (a student belongs to a classroom 💯🚀😉✨ you can check this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) to learn more how to implement it 


![image](https://github.com/ShaliniNayan/OOP-school-library/assets/58553711/4aff9a4c-fa9e-43f1-8e65-544a7607687a)